api/apps/api/src/modules/scenarios/marxan-run/marxan-run.controller.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Controller,
+  Get,
+  Header,
+  Param,
+  ParseUUIDPipe,
+  Res,
+  UseGuards,
+} from '@nestjs/common';
+import { ApiOkResponse, ApiOperation, ApiProduces } from '@nestjs/swagger';
+import { Response } from 'express';
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import { XApiGuard } from '@marxan-api/guards/x-api.guard';
+import { MarxanRunService } from './marxan-run.service';
+
+@UseGuards(XApiGuard)
```

Comment:
btw. (a note outside the scope of the PR)
x- prefixes are deprecated https://datatracker.ietf.org/doc/html/rfc6648
also for api-key we may use standard `Authorization` header